lib/raidenclient/RaidenClient.ts

```diff
@@ -309,6 +281,24 @@ class RaidenClient extends BaseClient {
     const res = await this.sendRequest(endpoint, 'PATCH', { state: 'settled' });
   }
 
+  /**
+   * Send a payment.
+   * @param target_address recipient of the payment
+   * @param token_address contract address of the token
+   * @param amount
+   * @param secret_hash optional; provide your own secret hash
+   */
+  public sendPayment = async (token_address: string, target_address: string, amount: number, secret_hash?: string): Promise<{}> => {
```

Comment:
One more thought, we'll want a common `sendPayment` interface for both lnd and raiden. Right now lnd has `sendPaymentSync`, and right now I'm looking at making that private and exposing a public `sendPayment` method with generic parameters that's defined on the base class. If you want to structure this similarly, maybe you could name this `tokenPayment` and make it private, then (possibly in a future PR) implement the public `sendPayment` method.

Reply:
I'll open a PR with a BaseClient `sendPayment` interface soon, right now I'm thinking of having it just accept a `SwapDeal` parameter and then have the logic to extract the necessary fields from it within.

Reply:
👍 - I rename the current implementation to `tokenPayment` and made it private.